bcipy/config.py

```diff
@@ -0,0 +1,25 @@
+DEFAULT_ENCODING = 'utf-8'
+
+# experiment configuration
+DEFAULT_EXPERIMENT_ID = 'default'
+EXPERIMENT_FILENAME = 'experiments.json'
+FIELD_FILENAME = 'fields.json'
+EXPERIMENT_DATA_FILENAME = 'experiment_data.json'
+DEFAULT_EXPERIMENT_PATH = '.bcipy/experiment/'
+DEFAULT_FIELD_PATH = '.bcipy/field/'
+
+DEFAULT_PARAMETERS_PATH = 'bcipy/parameters/parameters.json'
+
+# core data configuration
+DEFAULT_PARAMETER_FILENAME = 'parameters.json'

```

Comment:
`DEFAULT_PARAMETERS_PATH` could be defined using `DEFAULT_PARAMETER_FILE`:

    DEFAULT_PARAMETERS_PATH = f'bcipy/parameters/{DEFAULT_PARAMETER_FILENAME}'